### PR TITLE
Add WompMobile to providers

### PR DESCRIPTION
**Pull request by mattwomple**

### CODE REVIEW

File: pages/shared/data/tools.yaml [hunk 1]
```diff
@@ -466,3 +466,14 @@ providers:
       fit: contain
     formats:
       - email
+  - name: WompMobile
+    url: https://www.wompmobile.com
```
Comment (sebastianbenz) [inline pages/shared/data/tools.yaml]: Could you please link to a specific product/tool. Otherwise it's a bit confusing if you're listed under tools.

File: pages/shared/data/tools.yaml [hunk 2]
```diff
@@ -466,3 +466,14 @@ providers:
...
+    type: Development
+    description: WompMobile is an Accelerator Platform to develop, validate, launch and manage enterprise-grade AMP experiences.
```
Comment (sebastianbenz) [inline pages/shared/data/tools.yaml]: Can you rephrase in a way to focus on the tooling aspect?

File: pages/shared/data/tools.yaml (hunk 1 above)
Comment (mattwomple) [reply pages/shared/data/tools.yaml]: Updated URL to point to relevant tool: https://www.wompmobile.com/amp-experience-platform/

File: pages/shared/data/tools.yaml (hunk 2 above)
Comment (mattwomple) [reply pages/shared/data/tools.yaml]: Updated description to specifically mention use of tools: "Built-in tools monitor for errors, ensure validation, purge the AMP cache, and create Signed-Exchange and Server-Side-Rendered AMPs."
